src/routes/(console)/project-[region]-[project]/databases/database-[database]/collection-[collection]/attributes/string.svelte

```diff
@@ -42,20 +43,33 @@
 </script>
 
 <script lang="ts">
-    import { Selector } from '@appwrite.io/pink-svelte';
+    import { ActionMenu, Selector } from '@appwrite.io/pink-svelte';
     import { createConservative } from '$lib/helpers/stores';
     import { InputNumber, InputText, InputTextarea } from '$lib/elements/forms';
+    import { Popover, Layout, Tag, Typography, Link } from '@appwrite.io/pink-svelte';
+    import { organization } from '$lib/stores/organization';
+    import { BillingPlan } from '$lib/constants';
+    import { base } from '$app/paths';
 
     export let data: Partial<Models.AttributeString> = {
         required: false,
         size: 0,
         default: null,
-        array: false
+        array: false,
+        encrypt: false
     };
     export let editing = false;
 
     let savedDefault = data.default;
 
+    function handleEncryptedLabelClick(toggle: () => void) {
+        if ($organization?.billingPlan === BillingPlan.FREE) {
```

Comment:
Do we have access to the actual plan model here? Better to check if `databasesAllowEncrypt` is true, then if change which plans it's available on, we don't need any changes here